code/game/objects/items/theft_tools.dm

```diff
@@ -0,0 +1,209 @@
+	<li>Physical contact of any object with the sliver will dust the object, as well as yourself.</li>\
+	<li>Use the tongs to place the sliver into the provided container, which will take some time to seal</li>\
+	<li>Get the hell out before the crystal delaminates.</li>\
+	<li>???</li>"
+
+/obj/item/nuke_core/supermatter_sliver
+	name = "supermatter sliver"
+	desc = "A tiny, highly volatile sliver of a supermatter crystal. Do not handle without protection!"
+	icon_state = "supermatter_sliver"
+	item_state = "supermattersliver"
+	pulseicon = "supermatter_sliver_pulse"
+
+/obj/item/nuke_core/supermatter_sliver/attackby(obj/item/W, mob/living/user, params)
+	if(istype(W, /obj/item/weapon/hemostat/supermatter))
+		var/obj/item/weapon/hemostat/supermatter/tongs = W
+		if (tongs.sliver)
+			to_chat(user, "<span class='notice'>\The [tongs] is already holding a supermatter sliver!</span>")
+			return
+		src.forceMove(tongs)
+		tongs.sliver = src
+		tongs.icon_state = "supermatter_tongs_loaded"
+		to_chat(user, "<span class='notice'>You carefully pick up [src] with [tongs].</span>")
+	else if(istype(W, /obj/item/weapon/scalpel/supermatter) || istype(W, /obj/item/nuke_core_container/supermatter/)) // we don't want it to dust
+		return
+	else
+		to_chat(user, "<span class='notice'>As it touches \the [src], [W] bursts into dust!</span>")
+		radiation_pulse(get_turf(user), 1, 2, 10, 1)
+		qdel(W)
+
+/obj/item/nuke_core/supermatter_sliver/pickup(mob/living/user)
+	..()
+	if(!iscarbon(user))
+		return FALSE
+	var/mob/ded = user
+	to_chat(user, "<span class='warning'>You reach for the supermatter sliver. That was dumb.</span>")
+	radiation_pulse(get_turf(user), 2, 4, 50, 1)
+	ded.dust()
+
+/obj/item/nuke_core_container/supermatter
+	name = "supermatter receptacle"
+	desc = "A tiny receptacle that releases an inert freon mix upon sealing, allowing a sliver of a supermatter crystal to be safely stored.."
+	var/obj/item/nuke_core/supermatter_sliver/sliver = null
+
+/obj/item/nuke_core_container/supermatter/load(obj/item/weapon/hemostat/supermatter/T, mob/user)
+	if(!istype(T) || !T.sliver)
+		return 0
+	T.sliver.forceMove(src)
+	sliver = T.sliver
+	T.sliver = null
+	icon_state = "core_container_loaded"
+	to_chat(user, "<span class='warning'>Container is sealing...</span>")
+	addtimer(CALLBACK(src, .proc/seal), 50)
+	return 1
+
+/obj/item/nuke_core_container/supermatter/seal()
+	if(istype(sliver))
+		STOP_PROCESSING(SSobj, sliver)
+		icon_state = "core_container_sealed"
+		playsound(loc, 'sound/items/Deconstruct.ogg', 60, 1)
+		if(ismob(loc))
+			to_chat(loc, "<span class='warning'>[src] is permanently sealed, [sliver] is safely contained.</span>")
+
+/obj/item/nuke_core_container/supermatter/attackby(obj/item/weapon/hemostat/supermatter/tongs, mob/user)
+	if(istype(tongs))
+		if(!user.temporarilyRemoveItemFromInventory(tongs))
+			to_chat(user, "<span class='warning'>The [tongs] is stuck to your hand!</span>")
```

Comment:
How will the tongs being stuck to your hand prevent you from operating it while in your hand to release the sliver..?